geode-dunit/src/test/java/org/apache/geode/test/greplogs/LogConsumerTest.java

```diff
@@ -133,4 +133,14 @@ public void closeReturnsLineIfLineContainsSevereLogStatement() {
 
     assertThat(value).contains(line);
   }
+
+  @Test
+  public void closeReturnsLineIfLineContainsCurlies() {
```

Comment:
Maybe change the name to `closeReturnsLineIfLineContainsMalformedLog4jStatement`

Reply:
Done